packages/web-app-files/src/components/AppBar/ViewOptions.vue

```diff
@@ -83,7 +83,7 @@ export default {
     },
 
     toggleIcon() {
-      return this.sidebarClosed ? 'arrow-drop-left' : 'arrow-drop-right'
+      return this.sidebarClosed ? 'fill' : 'line'
```

Comment:
I think the other way around is better: if sidebar is closed, show `line`, otherwise show `fill`. This expresses the state, not the action. IMO that feels more natural.

Reply:
See suggested change on line 24